fw.json

```
@@ -7,7 +7,7 @@
         ["Flysky PL18", "pl18-"],
         ["Flysky PL18EV", "pl18ev-"],
         ["FrSky Horus X10", "x10-"],
-        ["FrSky Horus X10 Access", "x10-access-"],
+        ["FrSky Horus X10 Access", "x10express-"],
```

```suggestion
        ["FrSky Horus X10 Express", "x10express-"],
```
Probably want this to be in the user facing name also?